src/v2/controllers/vreplicaset_controller/proof/helper_invariants/predicate.rs

```diff
@@ -198,7 +198,9 @@ pub open spec fn every_delete_matching_pod_request_implies_at_after_delete_pod_s
     |s: ClusterState| {
         forall |msg: Message| #![trigger s.in_flight().contains(msg)] {
             let content = msg.content;
-            let key = content.get_delete_request().key;
+            let req = content.get_delete_request();
+            let req_rv = req.preconditions.unwrap().resource_version.unwrap();
+            let key = req.key;
```

Comment:
nit: change 201-203 does not seem necessary?

Reply:
Eliminating some of them, kept others.